<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoPanel</class>
 <widget class="QWidget" name="videoPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCameraViewfinder" name="viewFinder" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>491</width>
     <height>421</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>279</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="recordingSettingsButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Recording Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="recordButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Select video settings first</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Requested Frame Rate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="requestedFR">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Actual Frame Rate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="actualFR">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Frame Count</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="frameCount">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Timestamp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="timeStamp">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Resolution</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="resolutionEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
